frontend/src/layout/navigation-3000/sidebars/personsAndGroupsSidebarLogic.ts

```diff
@@ -59,7 +51,7 @@ export const personsAndGroupsSidebarLogic = kea<personsAndGroupsSidebarLogicType
                             } as BasicListItem
                         }),
                         loading: personsLoading,
-                        loadMore: persons.next ? () => actions.loadPersons(persons.next) : undefined,
+                        loadMore: persons.next ? () => actions.loadPersons(persons.next) : undefined, // FIXME
```

Comment:
fix what exactly 😅

Reply:
Well, `loadMore()` doesn't actually do anything right now – but I've already got a PR to fix this, so cleaned up